schema_utils.py

```diff
@@ -265,3 +265,12 @@ def is_regex(obj):
     def matches_regex(obj, regex):
         """Ensures that `obj` (a string) matches the given regex."""
         raise NotImplementedError
+
+    @staticmethod
+    def is_valid_email_list(obj):
+        """Ensures that `obj` (a list of strings) is a valid list of emails"""
+        def is_valid_email(email):
+            return '@' in email and not email.startswith('@') and \
```

Comment:
probably we can use a better validator here

Reply:
Also if the user sends duplicate emails, we can either filter at the time of sending or block them now.

Reply:
> Also if the user sends duplicate emails, we can either filter at the time of sending or block them now.

You can use `is_uniquified` validator for that purpose.